### DISCUSSION TITLE
Issue deploying REDCap instance

### DISCUSSION THREAD

**Discussion by jonny-ibdr** [microsoft/azure-redcap-paas#87] [Q&A]
Hi,

I ran into some issue deploying the bicep package, and I localized the issue to the `deploy.sh` failing to run. When I ran it manually, it would fail stating that the `unzip` package could not be found. This is (obviously) a required package because the redcap instance needs to be unzipped and extracted to the `site/wwwroot` folder.

## The Fix
- Manually ran `apt-get install unzip`
- Manually ran the deploy script: `bash site/repository/scripts/bash/deploy.sh`
- Waited roughly 10 minutes for this script to finish

Is this an issue with just my instance or does the linux web app now not ship with this package pre-installed?

Thanks

**Comment by SvenAelterman** [comment]
@jonny-ibdr,

There's a difference between running deploy.sh inside the container vs. what happens at deploy time. At deploy time, the deploy.sh is executed in an Oryx container, which does contain the unzip package.

I suspect the reason for the initial deployment failure was something else. Did you happen to take a look at the log file in the /home/site directory?
